apptail/instance.go

```diff
@@ -97,6 +119,12 @@ FORLOOP:
 				err = t.Wait()
 				break FORLOOP
 			}
+			currentOffset, err := t.Tell()
```

Comment:
The only way I see for `tail.TailFile()` to return an error is if the file doesn't exist; does trying to `t.Tell()` here ever make sense?  Also, does `t.Tell()` even return errors correctly? (It seems to return the uninitialized value of an `error`, and I have no idea what that is)